MapboxNavigation/InstructionsBannerViewLayout.swift

```diff
@@ -56,6 +56,8 @@ extension BaseInstructionsBannerView {
         self.separatorView = separatorView
         
         addTarget(self, action: #selector(BaseInstructionsBannerView.tappedInstructionsBanner(_:)), for: .touchUpInside)
```

Comment:
Tests are failing because this function still exists but was removed here: https://github.com/mapbox/mapbox-navigation-ios/pull/1150/files#diff-7bae0cf60f54997d0d5e86fe02b5cddbL60 and renamed to `draggedInstructionsBanner `

Reply:
Resolved ✅